### PR TITLE
Adding an first iteration of email template for report download

### PR DESCRIPTION
**Pull request by h-kataria**
This is an email template for Insights for AAP to send out a link to customers when Report is ready for download

https://issues.redhat.com/browse/AA-825

### CODE REVIEW

File: backend/src/main/resources/templates/Ansible/insightsEmailBody.html [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+      font-size: 18px;
+      text-align: center;
+    }
+
+    .rh-masthead__brand {
+      padding-top: 8px;
+    	padding-bottom: 8px;
+      background-color: #151515 !important;
+      background-repeat: repeat;
+      background-image: url(https://raw.githubusercontent.com/RedHatInsights/frontend-assets/master/src/email-assets/bg_151515.jpg);
```
Comment (josejulio) [inline backend/src/main/resources/templates/Ansible/insightsEmailBody.html:214]: I think you should be using this url

```suggestion
      background-image: url(https://console.redhat.com/apps/frontend-assets/email-assets/bg_151515.jpg);
```

File: backend/src/main/resources/templates/Ansible/insightsEmailBody.html [hunk 2]
```diff
@@ -0,0 +1,606 @@
...
+        height: 48px;
+      }
+
+      .rh-card.rh-m-set-height {
+        height: auto;
+      }
+    }
+
+    /* Dark Mode */
+		/* @media (prefers-color-scheme: dark) {
```
Comment (josejulio) [inline backend/src/main/resources/templates/Ansible/insightsEmailBody.html:468]: Dark mode is off?

**Review by josejulio — COMMENTED** [review]
This alone won't do much because:
 - We still need a mapping from your app to this template
   - https://github.com/RedHatInsights/notifications-backend/blob/master/backend/src/main/java/com/redhat/cloud/notifications/templates/Policies.java
   - and an update here: https://github.com/RedHatInsights/notifications-backend/blob/master/backend/src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateFactory.java#L22-L27
 - No dynamic content

Please let me know if you need help updating the java code.

**Comment by h-kataria** [comment]
@josejulio thanks for reviewing my Pr, I have made some suggested changes and addressed your comments. Please re-review when you get a chance.

File: backend/src/main/java/com/redhat/cloud/notifications/templates/Ansible.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
+package com.redhat.cloud.notifications.templates;
```
Comment (josejulio) [inline backend/src/main/java/com/redhat/cloud/notifications/templates/Ansible.java:1]: This file still looks like WIP. Let me know if you need help in here.

This file links the eventTypes to the actual html body and title for each email.

Comment (h-kataria) [reply backend/src/main/java/com/redhat/cloud/notifications/templates/Ansible.java:1]: @josejulio i will appreciate your help on this, I copied another file and made some changes to that.

File: backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/AnsibleEmailAggregator.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
+package com.redhat.cloud.notifications.processors.email.aggregators;
+
+import com.redhat.cloud.notifications.models.EmailAggregation;
+import io.vertx.core.json.JsonObject;
+
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
```
Comment (josejulio) [inline backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/AnsibleEmailAggregator.java:10]:
```suggestion
```

Almost there, removing unsed imports here

Comment (josejulio) [reply backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/AnsibleEmailAggregator.java:10]: I'll go ahead an commit the suggestion on your work.

**Review by josejulio — APPROVED** [review]
LGTM, waiting for tests

**Pull request merged**